signing_service/signing_service/tests/tests.py

```diff
@@ -21,6 +30,9 @@
 concent_ecc_keys = ECCx(None)
 (CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = concent_ecc_keys.raw_privkey, concent_ecc_keys.raw_pubkey
 
+signing_service_ecc_keys = ECCx(None)
+(SIGNING_SERVICE_PRIVATE_KEY, SIGNING_SERVICE_PUBLIC_KEY) = signing_service_ecc_keys.raw_privkey, signing_service_ecc_keys.raw_pubkey
+
 
 class SigningServiceMainTestCase(TestCase):
```

Comment:
quite frankly, here as well `unused_tcp_port` should be used (basically, everywhere where a real server is being used)